debezium-core/src/main/java/io/debezium/pipeline/notification/IncrementalSnapshotNotificationService.java

```diff
@@ -99,9 +99,7 @@ public <T extends DataCollectionId> void notifyAborted(IncrementalSnapshotContex
     public <T extends DataCollectionId> void notifyTableScanCompleted(IncrementalSnapshotContext<T> incrementalSnapshotContext, P partition, OffsetContext offsetContext,
                                                                       long totalRowsScanned, TableScanCompletionStatus status) {
 
-        String dataCollections = incrementalSnapshotContext.getDataCollections().stream().map(DataCollection::getId)
-                .map(DataCollectionId::identifier)
-                .collect(Collectors.joining(LIST_DELIMITER));
```

Comment:
What do you think to add a new field (maybe `SCANNED_COLLECTION`) as for in progress notification (`CURRENT_COLLECTION_IN_PROGRESS`) and leave `DATA_COLLECTIONS` unchanged?

Reply:
Sounds good!

Reply:
Thanks for the quick feedback! I'll make the changes.